### ISSUE TITLE
MD037 false positive with emojis containing odd number of underscore on same line

### ISSUE THREAD

**Issue by EnigmaTriton** [DavidAnson/markdownlint#408]
For example
```Markdown
:ballot_box_with_check: _Somewhat important task_
```
gives a warning for `_check: _` while the line is perfectly recognized by VSCode (and obviously github) as:

:ballot_box_with_check: _Somewhat important task_

I can work it out  using asterisks (or `:white_check_mark:` which contains an even number of underscores for this exact purpose) but I tend to use `_emphasis_` and `**strong**` to make it more obvious when editing source so this issue is slightly annoying.

On the other hand I'm not sure that emojis are part of the Common Mark specification so it might be an improvement request or even downright invalid request.

**Comment by DavidAnson** [comment]
Please see #324 for a discussion of this kind of issue.

**Comment by DavidAnson** [comment]
In this case, ignoring words with internal `_` would seem to help. I'll look for a way to do that without compromising the rule.

**Comment by EnigmaTriton** [comment]
Oh sorry, I searched for issues on MD037 but I probably forgot to remove the is:open part.

Much clearer, thanks and if you can take it into account anyway, then great. :+1:
